src/renderer/src/views/Home/Settings/ProjectConfig/MemberManagmentModal/DeviceInfoContent/ConfirmRemoveDevice.tsx

```diff
@@ -0,0 +1,93 @@
+import { Column, Row } from '@renderer/components/LayoutComponents'
+import ErrorIcon from '@mui/icons-material/Error'
+import { defineMessages, useIntl } from 'react-intl'
+
+import { useMapeoDeviceStore, useMapeoDeviceStoreAction } from '@renderer/hooks/stores/mapeoDeviceStore'
+import { Button } from '../../Button'
+import { theme } from '@renderer/theme'
+import { Typography } from '@mui/material'
+
+const m = defineMessages({
+  removeConfirmation: {
+    id: 'Settings.ProjectConfig.MemberManagement.ConfirmRemoveDevice.removeConfirmation',
+    defaultMessage: 'Are you sure you want to remove {name} ({id})?',
+  },
+  subtitle: {
+    id: 'Settings.ProjectConfig.MemberManagement.ConfirmRemoveDevice.subtitle',
+    defaultMessage:
+      'Their data will remain with the project but their device will be unable to sync or add new observations',
+  },
+  cancel: {
+    id: 'Settings.ProjectConfig.MemberManagement.ConfirmRemoveDevice.cancel',
+    defaultMessage: 'Cancel',
+  },
+  removeDevice: {
+    id: 'Settings.ProjectConfig.MemberManagement.ConfirmRemoveDevice.removeDevice',
+    defaultMessage: 'Remove Device',
+  },
+})
+
+type ConfirmRemoveDeviceProps = {
+  deviceId: string
+  closeModal: () => void
+  resetOnDelete: () => void
+}
+
+export const ConfirmRemoveDevice = ({ deviceId, closeModal, resetOnDelete }: ConfirmRemoveDeviceProps) => {
+  if (!deviceId) return null
+  const { formatMessage: t } = useIntl()
+  const name = useMapeoDeviceStore((store) => store.memberDevices[deviceId].name)
+  const removeDevice = useMapeoDeviceStoreAction().removeDeviceFromProject
+
+  function handleRemoveDevice() {
+    removeDevice(deviceId)
+    resetOnDelete()
+    closeModal()
+  }
+
+  return (
+    <Column height={'100%'} justifyContent={'space-between'}>
+      <Row padding={8} justifyContent="center" width={'100%'}>
+        <Column maxWidth="60%" alignItems="center" spacing={3}>
+          <Row>
+            <ErrorIcon style={{ fontSize: 100 }} />
+          </Row>
+          <Row>
+            <Typography variant="h1" textAlign="center" fontSize={24} fontWeight={500}>
+              {t(m.removeConfirmation, { name: name, id: deviceId })}
+            </Typography>
+          </Row>
+          <Row>
+            <Typography variant="h2" textAlign="center" fontSize={22} fontWeight={400}>
+              {t(m.subtitle)}
+            </Typography>
+          </Row>
+        </Column>
+      </Row>
+      <Row
+        justifyContent="space-between"
+        alignItems="center"
+        sx={{
+          paddingY: 2,
+          paddingX: 4,
+          borderTop: '1px solid',
+          borderColor: theme.grey.light,
+          width: '100%',
+        }}
+      >
+        <Button
+          noBorder
+          disableElevation
+          variant="contained"
+          sx={{ backgroundColor: theme.warningRed, color: theme.white }}
```

Comment:
i guess colors won't matter as much if we switch to black+white, but you'd want to provide a `color="warning"` prop so that your hover state is the correct color. ideally this would allow removing the `sx` completely, but my `Button` implementation has a minor bug where it [specifies the `backgroundColor`](https://github.com/digidem/mapeo-desktop-shell/blob/develop/src/renderer/src/views/Home/Settings/ProjectConfig/Button.tsx#L27). ideally would remove that line but then other areas will be affected and need to be updated, so better to do that as a separate PR.

in this case, the suggestion would make it so that the props are as follows:

```tsx
<Button color="warning" sx={{ backgroundColor: theme.warningRed }} .../>
```

before (on hover):

<img width="189" alt="image" src="https://user-images.githubusercontent.com/18542095/234896400-88461574-8831-4e26-8b74-c545065e3c37.png">

after (on hover):

<img width="186" alt="image" src="https://user-images.githubusercontent.com/18542095/234896454-9e17bf99-8f81-4c9b-a490-8c99fa550efc.png">